code/controllers/subsystem/processing/station.dm

```diff
@@ -21,6 +21,7 @@ PROCESSING_SUBSYSTEM_DEF(station)
 	#endif
 
 	announcer = new announcer() //Initialize the station's announcer datum
+	SSparallax.random_layer.apply_global_effects() //Apply station effects that parallax might have
```

Comment:
This seemed the most sensical place to put this? SSparallax inits before globals do, so we can't really change anything there. SSstation is already used for stuff like this with station traits (and I'm familiar with it)